### PR TITLE
Fix deployment

### PR DESCRIPTION
**Pull request by Keksoj**
- added instructions to the readme
- remove `maven.json` and replace with env variable instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,52 +39,46 @@ spring.data.mongodb.uri=${MONGODB_ADDON_URI}
...
-* run: `mvn spring-boot:run`
+In the environment variables, create `MAVEN_DEPLOY_GOAL` and set it to `spring-boot:run`
 
 ## Example
 
 Create new user
+
 ```bash
-curl -d '{"firstName":"DUVAL", "lastName":"Alexandre"}' -H "Content-Type: application/json" -X POST http://<appId>.cleverapps.io/users
+curl -d '{"firstName":"DUVAL", "lastName":"Alexandre"}' -H "Content-Type: application/json" -X POST https://<appId>.cleverapps.io/users
```
Comment (KannarFr) [inline README.md]: Robert Cloud?

Comment (Keksoj) [reply README.md]: it was here before I swear :-D

Comment (KannarFr) [reply README.md]: I mean, can you apply it here too. Then auto-merge it if you need it for the demo tomorrow otherwise I'll do it.

**Comment by Keksoj** [comment]
poke @KannarFr like so?

**Pull request merged**